src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_ITypeOfExpression.cs

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void TestTypeOfExpression()
+        {
+            string source = @"
+using System;
+
+class C
+{
+    void M(Type t)
+    {
+        t = /*<bind>*/typeof(int)/*</bind>*/;
```

Comment:
Will VB's `GetType` be represented as a typeof IOperation? If not, do we know how it will be represented?

Reply:
I thought we had a separate API for `GetType`, but apparently we do not have it. I agree that it makes sense for `GetType` to be represented as an `ITypeOfExpression`. Tagging @dotnet/analyzer-ioperation for confirmation.